console/src/assets/i18n/ja.json

```
@@ -0,0 +1,2095 @@
+    "TABLE": {
+      "TOTAL": "エントリ合計：",
+      "SELECTION": "選択された要素",
+      "DEACTIVATE": "プロジェクトの非アクティブ化",
+      "ACTIVATE": "プロジェクトのアクティブ化",
+      "DELETE": "プロジェクトの削除",
+      "ORGNAME": "組織名",
+      "ORGDOMAIN": "組織ドメイン",
+      "STATE": "ステータス",
+      "TYPE": "タイプ",
+      "CREATIONDATE": "作成日",
+      "CHANGEDATE": "最終更新日",
+      "RESOURCEOWNER": "所有者",
+      "SHOWTABLE": "テーブルを表示する",
+      "SHOWGRID": "グリッドを表示する",
+      "EMPTY": "プロジェクトは見つかりません"
+    },
+    "TOAST": {
+      "MEMBERREMOVED": "マネージャーが削除されました。",
+      "MEMBERSADDED": "マネージャーが追加されました。",
+      "MEMBERADDED": "マネージャーが追加されました。",
+      "MEMBERCHANGED": "マネージャーが変更されました。",
+      "ROLESCREATED": "ロールが作成されました。",
+      "ROLEREMOVED": "ロールが削除されました。",
+      "ROLECHANGED": "ロールが変更されました。",
+      "REACTIVATED": "アクティブになりました。",
+      "DEACTIVATED": "非アクティブになりました。",
+      "CREATED": "プロジェクトが作成されました。",
+      "UPDATED": "プロジェクトが変更されました。",
+      "GRANTUPDATED": "グラントが変更されました。",
+      "DELETED": "プロジェクトが削除されました。"
+    }
+  },
+  "ROLES": {
+    "DIALOG": {
+      "DELETE_TITLE": "ロールの削除",
+      "DELETE_DESCRIPTION": "ロールを削除しようとしています。本当によろしいですか？"
+    }
+  },
+  "NEXTSTEPS": {
+    "TITLE": "次のステップ"
+  },
+  "IDP": {
+    "LIST": {
+      "TITLE": "IDプロバイダー",
+      "DESCRIPTION": "IDプロバイダーの構成を管理し、ログイン設定で有効にすることができます。",
+      "ACTIVETITLE": "アクティブなIDプロバイダー"
+    },
+    "CREATE": {
+      "TITLE": "プロバイダーの追加",
+      "DESCRIPTION": "以下のプロバイダーから1つ以上選択してください。",
+      "STEPPERTITLE": "プロバイダーの作成",
+      "OIDC": {
+        "TITLE": "OIDCプロバイダー",
+        "DESCRIPTION": "OIDCプロバイダーに必要なデータを入力してください。"
+      },
+      "OAUTH": {
+        "TITLE": "OAuthプロバイダー",
+        "DESCRIPTION": "OAuthプロバイダーに必要なデータを入力してください。"
+      },
+      "JWT": {
+        "TITLE": "JWTプロバイダー",
+        "DESCRIPTION": "JWTプロバイダーに必要なデータを入力してください。"
+      },
+      "GOOGLE": {
+        "TITLE": "Googleプロバイダー",
+        "DESCRIPTION": "Google Identity Providerのクレデンシャルを入力してください。"
+      },
+      "GITHUBES": {
+        "TITLE": "GitHub Enterprise Serverプロバイダー",
+        "DESCRIPTION": "GitHub Enterprise Server IDプロバイダーのクレデンシャルを入力してください。"
+      },
+      "GITHUB": {
+        "TITLE": "Githubプロバイダー",
+        "DESCRIPTION": "GitHub IDプロバイダーのクレデンシャルを入力してください。"
+      }
+    },
+    "DETAIL": {
+      "TITLE": "IDプロバイダー",
+      "DESCRIPTION": "プロバイダーの構成を更新する",
+      "DATECREATED": "作成日",
+      "DATECHANGED": "更新日"
+    },
+    "OPTIONS": {
+      "ISAUTOCREATION": "自動作成",
+      "ISAUTOCREATION_DESC": "選択した場合、アカウントがまだ存在しない場合はアカウントが作成されます。",
+      "ISAUTOUPDATE": "自動更新",
+      "ISAUTOUPDATE_DESC": "選択した場合、アカウントは再認証時に更新されます。",
+      "ISCREATIONALLOWED": "アカウント作成を許可",
+      "ISCREATIONALLOWED_DESC": "アカウントを作成できるかどうかを決めます。",
+      "ISLINKINGALLOWED": "アカウントリンクを許可",
+      "ISLINKINGALLOWED_DESC": "IDを既存のアカウントにリンクできるかどうかを決めます。"
+    },
+    "OWNERTYPES": {
+      "0": "不明",
+      "1": "インスタンス",
+      "2": "組織"
+    },
+    "STATES": {
+      "1": "アクティブ",
+      "2": "非アクティブ"
+    },
```

```suggestion
    },
    "AZUREADTENANTTYPES": {
      "0": "Common",
      "1": "Organizations",
      "2": "Customers"
    },
    "AZUREADTENANTTYPE": "Tenant Type",
    "AZUREADTENANTID": "Tenant ID",
    "EMAILVERIFIED": "Email verified",
```

this was also added in the meantime